redesign.user.js

```diff
@@ -316,6 +321,11 @@ Viele Grüße
                         trans: 'Transgender',
                     },
                 },
+                quickRoleChange: {
+                    name: 'Schneller Rollenwechsel',
+                    description:
+                        'Ermöglicht es, die Rolle ähnlich zu der Sprache zu ändern (mit den passenden Berechtigungen).',
```

Comment:
Irgendwie finde ich das etwas unklar. Können wir das vielleicht etwas spezifizieren?
So etwas in die Richtung "Ermöglicht es, die Betrachtung mit einer anderen Rolle direkt aus dem Profil-Dropdown aus zu ändern." evtl.?

Reply:
Resolved by [9b900b6](https://github.com/jxn-30/better-moodle/pull/215/commits/9b900b6eb5a3790fa8b324254f3517ace0564a2a)